scripts/release/generate_history_notes.py

```diff
@@ -26,8 +26,38 @@
 history_line_breaker = '==============='
 history_notes = {}
 
+# key is lower case and removed spaces of a component
+# value is the recommended format of a component
+customized_dict = {
+    "privatedns": "Private DNS",
+    "appconfig": "App Config",
+}
+
+# This dict will be filled with all historical compenents.
+# key is lower case and removed spaces of a component
+# value is the recommended format of a component, when there're multiple formats,
+# the one with spaces will be picked, i.e. pick 'Key Vault' over 'KeyVault'.
+# If the key also exists in customized_dict, the value will be overwritten with customized_dict[key].
+component_dict = {}
```

Comment:
Is component a synonym for module is this context?

Reply:
It includes module as the majority case but could also be something like `Packaging`, `Test`.